prometheus/base/alerting/prometheus.argocd_alerts.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-argocd-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: argocd_alerts
+    rules:
+    - alert: DegradedArgocdApp
+      expr: max_over_time(argocd_app_info{health_status="Degraded", dest_namespace!~".+-tenant"}[5m]) == 1
```

Comment:
One of the acceptance criteria is to ensure the degraded component's `service` label is included in the alert. This metric always returns the value of `gitops-service-argocd-metrics` no matter which `dest_namespace` is affected. Have you tried to solve this?

Reply:
the summary and the description return the affected app's name

Reply:
The `name` label looks something like `gitopsdepl-0c63a1fd-fbd3-41d4-ba02-50290ad8681e` which isn't very descriptive and probably not useful when we eventually need to route this alert. The `service` label is consistently available in the other alert rules so it'd be nice to conform to that standard, if possible.